cves/kernel/CVE-2013-2017.yml

```diff
@@ -1,288 +1,196 @@
+    typography, unicode, or various other features.
+
+    Answer should be true or false
+    Write a note about how you came to the conclusions you did, regardless of
+    what your answer was.
+vccs:
+  - commit: 445409602c09219767c06497c0dc2285eac244ed
+    note: Discovered automatically by archeogit.
+  - commit: 38d408152a86598a50680a82fe3353b506630409
+    note: Discovered automatically by archeogit.
+fixes:
+  - commit: 6ec82562ffc6f297d0de36d65776cff8e5704867
+    note: Manually confirmed
+vouch:
+  note: |
+    Both Eric Dumazet and David S. Miller are on the fix commit as signing off on the fix.
+  answer: true
+  question: >
+    Was there any part of the fix that involved one person vouching for
+
+    another's work?
+
+
+    This can include:
+      * signing off on a commit message
+      * mentioning a discussion with a colleague checking the work
+      * upvoting a solution on a pull request
+
+    Answer must be true or false.
+
+    Write a note about how you came to the conclusions you did, regardless of what your answer was.
 bounty:
   amt:
-  announced:
   url:
-reviews: []
-bugs_instructions: |
-  What bugs are involved in this vulnerability?
-
-  Please list bug IDs to https://bugzilla.kernel.org/
+  announced:
+lessons:
+  yagni:
+    note:
+    applies: false
+  question: |
+    Are there any common lessons we have learned from class that apply to this
+    vulnerability? In other words, could this vulnerability serve as an example
+    of one of those lessons?
 
-  Bug ID's can appear in several places:
-    * Mentioned in commit messages
-    * Mentioned in mailing list discussions
-    * References from NVD entry
-    * Various other places
-bugs: []
-fixes_instructions: |
-  Please put the commit hash in "commit" below.
+    Leave "applies" blank or put false if you did not see that lesson (you do
+    not need to put a reason). Put "true" if you feel the lesson applies and put
+    a quick explanation of how it applies.
 
-  This must be a git commit hash from the systemd source repo, a  40-character
-  hexademical string/
+    Don't feel the need to claim that ALL of these apply, but it's pretty likely
+    that one or two of them apply.
 
-  Place any notes you would like to make in the notes field.
-fixes:
-- commit:
-  note:
-- commit:
-  note:
-- commit: 6ec82562ffc6f297d0de36d65776cff8e5704867
+    If you think of another lesson we covered in class that applies here, feel
+    free to give it a small name and add one in the same format as these.
+  serial_killer:
+    note:
+    applies: false
+  complex_inputs:
+    note:
+    applies: false
+  distrust_input:
+    note:
+    applies: false
+  least_privilege:
+    note:
+    applies: false
+  native_wrappers:
+    note:
+    applies: false
+  defense_in_depth:
+    note:
+    applies: false
+  secure_by_default:
+    note:
+    applies: false
+  environment_variables:
+    note:
+    applies: false
+  security_by_obscurity:
```

Comment:
Look into this one, as I believe it related to your vulnerability. Security by obscurity.

Reply:
I'm not too sure security by obscurity applies here as I don't think I saw anything that purposefully hid data just to make secure. Unless I have my understanding of that lesson wrong.